### PR TITLE
[generator] Saving LineStrings to file.

### PR DESCRIPTION
**Pull request by maksimandrianov**

### CODE REVIEW

File: generator/metalines_builder.hpp [hunk 1]
```diff
@@ -37,7 +40,29 @@ class LineString
   uint64_t GetStart() const { return m_start; }
   uint64_t GetEnd() const { return m_end; }
 
+  template <typename T>
+  void Serialize(T & w)
+  {
+    WriteVarUint(w, m_start);
+    WriteVarUint(w, m_end);
+    w.Write(&m_oneway, sizeof(m_oneway));
```
Comment (mpimenov) [inline generator/metalines_builder.hpp:48]: WriteToSink?

**Review by mpimenov — APPROVED** [review]
LGTM

File: generator/metalines_builder.hpp (hunk 1 above)
Comment (maksimandrianov) [reply generator/metalines_builder.hpp]: ++

File: generator/metalines_builder.cpp [hunk 2]
```diff
@@ -187,11 +190,23 @@ void MetalinesBuilder::CollectFeature(FeatureBuilder const & feature, OsmElement
...
-  m_data.emplace(key, std::make_shared<LineString>(element));
+  WriteVarUint(*m_writer, key);
+  LineString(element).Serialize(*m_writer);
 }
 
+void MetalinesBuilder::Finish() { m_writer.reset(); }
+
 void MetalinesBuilder::Save()
 {
+  std::unordered_multimap<size_t, std::shared_ptr<LineString>> m_data;
```
Comment (tatiana-yan) [inline generator/metalines_builder.cpp:201]: s/m_data/data ?

**Review by tatiana-yan — APPROVED** [review]
LGTM after fix

File: generator/metalines_builder.cpp (hunk 2 above)
Comment (maksimandrianov) [reply generator/metalines_builder.cpp:201]: ++ keyToLineString

**Pull request merged**